presto-mysql/src/main/java/com/facebook/presto/plugin/mysql/MySqlClient.java

```diff
@@ -173,6 +191,10 @@ protected String toSqlType(Type type)
             // In order to preserve microsecond information for TIMESTAMP_MICROSECONDS
             return "datetime(6)";
         }
+        if (type.getTypeSignature().getBase().equalsIgnoreCase("Geometry")) {
```

Comment:
```suggestion
        if (GEOMETRY.equals(type)) {
```
Use `com.facebook.presto.geospatial.type.GEOMETRY` 
You'll need to add presto-main-base as a dependency

Reply:
Applied